op-mode-definitions/ipoe-server.xml

```diff
@@ -1,5 +1,41 @@
 <?xml version="1.0"?>
 <interfaceDefinition>
+  <node name="reset">
+    <children>
+      <node name="ipoe-server">
+        <properties>
+          <help>Clear ipoe-server sessions or process</help>
+        </properties>
+        <children>
+          <node name="session">
+            <properties>
+              <help>Clear ipoe-server session</help>
+            </properties>
+            <children>
+              <tagNode name="username">
```

Comment:
Is there a reason for using tagNode instead of leafnode?

Reply:
Hello! no reason...

Reply:
Can you use then <leafNode> please?
Also, since it appears that you are using the IPoE service within VyOS, I would be very interested on feedback and user scenarios.